docs/pywbemclicommands.rst

```diff
@@ -77,42 +86,61 @@ See :ref:`pywbemcli class associators --help` for details.
 Class references command
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``class references`` command retrieves association classes or class names for a
-class defined by the CLASSNAME argument in the default namespace or the
-namespace defined with this command displayed in the defined format. If
-successful it displays the classes/classnames in the
-:term:`CIM object output formats` (see :ref:`Output formats`).
-If unsuccesful it returns an  exception. This command
-returns the class level references,not the instance references. The
-:ref:`Instance command group` includes a corresponding instance references
-operation:
+The ``class references`` command lists the CIM classes that reference
+the specified source class.
+
+The name of the source class is specified with the ``CLASSNAME`` argument,
+and the class is looked up in the namespace specified with the
```

Comment:
looked up again

Reply:
Discussion with Karl: Instead of "looked up" he suggests "acquires" or "acquires from", or even "gets" or "gets from" would be ok.

Reply:
Changed the sentence to:
"The class is named with the ``CLASSNAME`` argument and is in the namespace specified with the ``-namespace``/``-n`` command option or in the default namespace of the connection."